pkg/render/fluentd.go

```diff
@@ -94,7 +94,9 @@ const (
 	FluentdNodeName        = "fluentd-node"
 	fluentdNodeWindowsName = "fluentd-node-windows"
 
-	eksLogForwarderName = "eks-log-forwarder"
+	EKSNodeName                  = "eks-log-forwarder-node"
```

Comment:
I think we can just use `EKSLogForwarderName` here - the `NodeName` suffix is mostly used for daemonset components that run on every node (e.g., calico-node, fluentd-node, etc) so we don't need to copy that pattern here.